.github/workflows/workflow_test.yaml

```diff
@@ -0,0 +1,8 @@
+name: Workflow test
+
+on:
```

Comment:
I don't think you need this file. It is possible to define multiple events triggering a pipeline so you can add `pull_request` in `./.github/workflows/test.yaml`

Reply:
My initial thought process was to have separate yaml and calling it as how the end-users would use it, but second thought, i think this way does sound better :) Thanks for the feedback!

Reply:
Hello Arturo, an update since last comment,
I've still opted for having a separate file for the following reasons:
1. We can simulate external user's behaviors
2. Extensible inputs with different tox files / other future inputs that we might be testing(e.g. Dockerfile, shellscripts) with different scenarios (e.g. failing tox case)

:)